mlflow/R/mlflow/R/tracking-globals.R

```diff
@@ -1,5 +1,9 @@
+#' @include tracking-observer.R
+NULL
+
 mlflow_set_active_run_id <- function(run_id) {
```

Comment:
Out of curiosity, why do we register the callback on `set_active_run_id` rather than on `mlflow_start_run`?